docs/decisions/0006-xapi-backend-replacement-enterprise_uuid.rst

```diff
@@ -0,0 +1,24 @@
+Adding enterprise_uuid in course enrollment event
+--------------------------------------------------
+
+Status
+======
+
+Pending
+
+Context
+=======
+
+Event_routing_backend application needs ``enterprise_uuid`` to send enrollment events to the relevant enterprise customers. At present, ``enterprise_uuid`` is not included in the course enrollment events generated by edx enterprise.
+
+Decision
+=======
+
+``enterprise_uuid`` will be added in the course enrollment information sent by edx platform and will be added to the context of ``EVENT_NAME_ENROLLMENT_ACTIVATED`` emitted by edx platform.
+
+
+Consequences
+=======
+
+1. ``enterprise_uuid`` will be present in the context of ``EVENT_NAME_ENROLLMENT_ACTIVATED``.
```

Comment:
@ayub02 I think we can have only this one.
``enterprise_uuid`` will be present in the context of ``EVENT_NAME_ENROLLMENT_ACTIVATED`` for enterprise enrollments.
In case of non enterprise enrollments event would not have ``enterprise_uuid`` in context as it is now.